include/swift/AST/TypeCheckRequests.h

```diff
@@ -1333,17 +1333,17 @@ class GetDistributedActorInvocationDecoderRequest :
 
 /// Obtain the method that could be used to decode argument values passed
 /// to a particular actor invocation type.
-class GetDistributedActorArgumentDecodingMethodRequest :
-  public SimpleRequest<GetDistributedActorArgumentDecodingMethodRequest,
-                       FuncDecl *(NominalTypeDecl *),
+class GetDistributedActorConcreteArgumentDecodingMethodRequest :
+  public SimpleRequest<GetDistributedActorConcreteArgumentDecodingMethodRequest,
+                       FuncDecl *(DeclContext *),
```

Comment:
I think this is a bit too drastic, can't we ask the callers to provide us with a type instead?

Reply:
fixed ok